orchestration/dagster_orchestration/hca_orchestration/tests/e2e/test_updated_staging_area.py

```diff
@@ -0,0 +1,73 @@
+from datetime import datetime
+import pytest
+from dagster import execute_pipeline
+from google.cloud.bigquery import QueryJobConfig, Client
+
+from hca_orchestration.pipelines import load_hca
+
+
+@pytest.mark.e2e
+def test_load_updated_staging_area(
```

Comment:
This is the e2e updates test. It fires off an import of the same staging data as `test_load_hca.py`, then fires off an import of a staging area with an updated entity I created. We may want to consider rolling the assertions from `test_load_hca` into the first section of this test to avoid running the same slow-ish import twice, but I think this is ok for now.